README.md

```diff
@@ -76,6 +76,9 @@ tutorial.
 *  [Replicated Region](replicated/README.md)
 *  [Partitioned Region](partitioned/README.md)
 *  [Put Multiple Values at Once](putall/README.md)
+*  [Functions](functions/README.md)
+*  [Put Multiple Values at Once](putall/README.md)
+*  [Functions](functions/README.md)
```

Comment:
There are now 2 links to the functions example, and there are 2 links to the put all example.